src/components/SupportingPartners/index.stories.js

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+
+import Index from ".";
+
+export default {
+  title: "Components/Supporting Partners",
+  argTypes: {
+    title: {
+      control: {
+        type: "text",
+      },
+      partners: {
+        control: {
+          type: "array",
```

Comment:
```suggestion
          type: "object",
```
Control type for array type is object https://storybook.js.org/docs/react/essentials/controls#annotation